pulsar-broker/src/main/java/org/apache/pulsar/broker/admin/AdminResource.java

```diff
@@ -624,7 +624,12 @@ && pulsar().getConfig().isCreateTopicToRemoteClusterForReplication()) {
                     asyncResponse.resume(Response.noContent().build());
                 })
                 .exceptionally(ex -> {
-                    log.error("[{}] Failed to create partitioned topic {}", clientAppId(), topicName, ex);
+                    if (AdminResource.isConflictException(ex)) {
+                        log.warn("[{}] Failed to create partitioned topic {}: {}", clientAppId(), topicName,
```

Comment:
When encountering a resource NotFound or Exist exception, maybe just printing an info level log is better?

Reply:
Make sense. Actually, I suggest we don't even need to log these errors because they are not broker errors and can be reported to the client.